docs/book/src/developer/tilt.md

```diff
@@ -169,6 +169,15 @@ kustomize_substitutions:
   GCP_B64ENCODED_CREDENTIALS: "your credentials here"
 ```
 
+{{#/tab }}
+{{#tab vSphere}}
+
+```yaml
+kustomize_substitutions:
+  "VSPHERE_USERNAME": "administrator@vsphere.local",
+  "VSPHERE_PASSWORD": "Admin123",
```

Comment:
```suggestion
  VSPHERE_USERNAME: "administrator@vsphere.local"
  VSPHERE_PASSWORD: "Admin123"
```
Up to you if you want to drop the quotes around the keys, but the commas have to go :)